scripts/public-reviews/verify-snapshot-pr.cjs

```
@@ -0,0 +1,1049 @@
+    for (const temporaryRef of [candidateRef, baseRef]) {
+      try {
+        deleteCandidateRef(context.repositoryRoot, temporaryRef, status);
+      } catch (error) {
+        cleanupErrors.push(error);
+      }
+    }
+    if (primaryError && cleanupErrors.length > 0) {
+      throw new AggregateError(
+        [primaryError, ...cleanupErrors],
+        "Snapshot verification and trusted cleanup both failed."
+      );
```

Biome (2.5.3)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)